extension-impl/remoting-http/src/main/java/com/alipay/sofa/rpc/transport/http/SslContextBuilder.java

```diff
@@ -36,15 +36,16 @@
 public class SslContextBuilder {
 
     public static final boolean SSL = System.getProperty("ssl") != null;
+    public static final String CERTIFICATE_PATH = System.getProperty("certificate_path");
+    public static final String PRIVATE_KEY_PATH = System.getProperty("private_key_path");
```

Comment:
`certificate_path` and `private_key_path` use constants.

Reply:
do you mean modifing the codes as below?

 public static final String ProPerty_CERTIFICATE_PATH = "certificate_path";
 public static final String ProPerty_PRIVATE_KEY_PATH = "private_key_path";
 public static final String CERTIFICATE_PATH = System.getProperty(ProPerty_CERTIFICATE_PATH);
 public static final String PRIVATE_KEY_PATH = System.getProperty(ProPerty_PRIVATE_KEY_PATH);

Reply:
yes

Reply:
`public static final` field need use UPPERCASE